### PR TITLE
HPCC-29378 Document Pipe Restrictions

### PR DESCRIPTION
**Pull request by JamesDeFabia**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by g-pan — APPROVED** [review]
approved

**Comment by ghalliday** [comment]
@JamesDeFabia please fill in the checkboxes

**Comment by JamesDeFabia** [comment]
@ghalliday checkboxes completed now

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 1]
```diff
@@ -319,6 +319,125 @@ helm upgrade mycluster hpcc/hpcc -f tworoxies.yaml -f twothors.yaml
...
+        </listitem>
+      </orderedlist>
+
+      <para><emphasis role="bold">Deploy using the new custom configuration
+      chart.</emphasis></para>
+
+      <orderedlist>
+        <listitem>
+          <para>Open a terminal and navigate to the folder where you saved the
+          twothors.yaml file.</para>
```
Comment (jakesmith) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:405]: I think this was supposed to say "thorWithPipe.yaml file"

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 2]
```diff
@@ -319,6 +319,125 @@ helm upgrade mycluster hpcc/hpcc -f tworoxies.yaml -f twothors.yaml
...
+        <listitem>
+          <para>After you confirm that your deployment is running, submit a
+          job that uses a PIPE action and specifies one of the programs you
+          specified. </para>
+
+          <para><emphasis role="bold">Note:</emphasis> If the job is too
+          simple, it will execute on hThor instead of Thor and this example
+          doesn't enable Pipe programs on hThor. </para>
+
+          <para>You create another yaml file to allow Pipe Programs on ECL
```
Comment (jakesmith) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:431]: You *can* create

**Review by jakesmith — COMMENTED** [review]
@JamesDeFabia - a couple of minors..

**Comment by JamesDeFabia** [comment]
@jakesmith Edits made, good catches

**Review by jakesmith — APPROVED** [review]
@JamesDeFabia - looks good.

**Pull request merged**